web/app/dashboard/layout.tsx

```diff
@@ -4,7 +4,7 @@ export default async function DashboardLayout({
   children: React.ReactNode;
 }) {
   return (
-    <div className='flex w-full flex-1 flex-col items-center justify-start px-3'>
+    <div className='flex w-full flex-1 flex-col items-center justify-start'>
```

Comment:
le padding c'etait pour la version mobile, sinon c'est bord à bord

```suggestion
    <div className='flex w-full flex-1 flex-col items-center justify-start px-3 sm:px-0'>
```